packages/wast-parser/test/fixtures/export/global/expected.json

```diff
@@ -37,9 +37,8 @@
           "descr": {
             "type": "Global",
             "id": {
-              "type": "NumberLiteral",
-              "value": 0,
-              "raw": "0"
+              "type": "Identifier",
+              "value": "global_0"
```

Comment:
We should preserve the `raw` value because it will be printed as `global_0` otherwise.

Reply:
Good point, I presume the raw values are used for round tripping back to wasm / wat?

Reply:
I'm not sure what do you mean by "round tripping back to wasm / wat" but we already have deterministic transformation for Identifier ⟶ Indice and the inverse. We can transform for wasm as well as for wast.

`raw` is just for printing I think.

Reply:
I think this is a bigger change, so I've pulled it out into this issue for clarification https://github.com/xtuc/webassemblyjs/issues/228